Management/src/ActuatorWeb/appsettings.json

```diff
@@ -0,0 +1,52 @@
+{
+  // Steeltoe: Add schema to get auto-completion.
+  "$schema": "https://steeltoe.io/schema/latest/schema.json",
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning",
+      "Steeltoe.Samples.ActuatorWeb.Pages.WeatherModel": "Debug"
+    }
+  },
+  "AllowedHosts": "*",
+  // Steeltoe management endpoint configuration.
+  "Management": {
+    "Endpoints": {
+      // Only show health details when the user has a scope claim with value 'actuator.read'.
+      "Health": {
+        "ShowDetails": "WhenAuthorized",
+        "Claim": {
+          "Type": "scope",
+          "Value": "actuator.read"
+        }
+      },
+      // Include all available properties in HttpExchanges.
+      "HttpExchanges": {
+        "AddAuthType": true,
+        "AddParameters": true,
+        "AddPathInfo": true,
+        "AddQueryString": true,
+        "AddRemoteAddress": true,
+        "AddRequestHeaders": true,
+        "AddResponseHeaders": true,
+        "AddSessionId": true,
+        "AddTimeTaken": true,
+        "AddUserPrincipal": true
+      },
+      // Configure local time actuator.
+      "LocalTime": {
+        "Format": "dd-MM-yyyy HH:mm:ss,fff"
+      },
+      // Configure endpoints to be exposed on a dedicated port.
+      "Port": "8090"
+      // This property would cause HTTPS to be required for management endpoints.
+      // "SslEnabled": true
+    }
```

Comment:
I don't consider it helpful to have commented-out settings without documenting the impact and providing a working experience. When I uncomment this, the endpoints in .http file fail (infinite timeout) because there's no HTTPS environment defined and the actuators don't perform auto-redirect to https. It also breaks SBA. Given these, I didn't test any further.

Consider defining a separate launch profile to demonstrate using a dedicated actuator port with SSL, and ensure everything works or limitations are documented. Otherwise, it should just be left out.

Reply:
Yeah, there are several fragments required to make it all work and we can't offer a complete solution without enhancing the SBA image. I knew that in advance, that's why I left the bits commented out - I honestly did not expect you to start uncommenting them and testing with incomplete configurations. 

I'll stash these changes and we can come back to them if we get around to the [SBA image enhancement](https://github.com/SteeltoeOSS/Dockerfiles/issues/40)